host/packages/cuttlefish-integration/etc/ssh/sshd_config.cuttlefish

```diff
@@ -0,0 +1,123 @@
+#	$OpenBSD: sshd_config,v 1.100 2016/08/15 12:32:04 naddy Exp $
+
+# This is the sshd server system-wide configuration file.  See
+# sshd_config(5) for more information.
+
+# This sshd was compiled with PATH=/usr/bin:/bin:/usr/sbin:/sbin
+
+# The strategy used for options in the default sshd_config shipped with
+# OpenSSH is to specify options with their default value where
+# possible, but leave them commented.  Uncommented options override the
+# default value.
+
+#Port 22
+#AddressFamily any
+#ListenAddress 0.0.0.0
+#ListenAddress ::
+
+#HostKey /etc/ssh/ssh_host_rsa_key
+#HostKey /etc/ssh/ssh_host_ecdsa_key
+#HostKey /etc/ssh/ssh_host_ed25519_key
+
+# Ciphers and keying
+#RekeyLimit default none
+
+# Logging
+SyslogFacility LOCAL1
+LogLevel DEBUG1
```

Comment:
What do you think of using a transform instead of a displace here? It's hard to tell from this file what's special about cuttlefish-integration and what were the existing options set in the file. With a transform, only the added lines will have to be kept in the android-cuttlefish repository. I think the transform would be something like

/etc/ssh/sshd_config cat - <(echo "SyslogFacility LOCAL1") <(echo "LogLevel DEBUG1")

The dash adds stdin, then the two echo statements add the extra lines.

http://manpages.ubuntu.com/manpages/bionic/man1/dh_configpackage.1.html

The rest of the file could then be removed. That avoids permanently fixing the sshd_config when other parts of debian may want to change it later.

Reply:
Thanks for the suggestion. That approach won't work because /etc/ssh/sshd_config doesn't appear in dpkg -L, and the transform checks for this before packaging.

I did something similar in the postinstll script.